src/infra/raft/v2/RaftCore.cpp

```diff
@@ -164,8 +164,10 @@ void RaftCore::initService(const INIReader &iniReader) {
   /// init RaftClient
   for (const auto &p : mPeers) {
     auto &peer = p.second;
+    grpc::ChannelArguments chArgs;
+    chArgs.SetMaxReceiveMessageSize(INT_MAX);
```

Comment:
where is `INT_MAX` defined?

Reply:
be cautious that we made the promise that INT_MAX is supported, we'd better make sure it works as promised.